src/cson.coffee

```diff
@@ -60,6 +60,7 @@ parseContents = (objectPath, cachePath, contents, callback) ->
   try
     object = parseObject(objectPath, contents)
   catch parseError
+    parseError.message = "#{objectPath}: #{parseError.message}"
```

Comment:
It wouldn't appear in the stack trace, but is helpful when debugging and is similar to `error.code` that is often set.

Here is an example:

```sh
$ node
> try { require('fs').readFileSync('/does/not/exist') } catch (error) { console.log(error.path); console.log(error.message) }
/does/not/exist
ENOENT, no such file or directory '/does/not/exist'
```